nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,106 @@
+                  argstr='-i %s', xor=['feat_dir'], 
+                  desc='volume to be denoised')
+    out_dir = Directory('out', mandatory=True,
+                       argstr='-o %s',  
+                       desc='output directory')
+    mask = File(exists=True, argstr='-m %s', xor=['feat_dir'], 
+               desc='path/name volume mask')
+    dim = traits.Int(argstr='-dim %d',
+                    desc='Dimensionality reduction when running '
+                    'MELODIC (defualt is automatic estimation)')
+    TR = traits.Float(argstr='-tr %.3f',
+                     desc='TR in seconds. If this is not specified '
+                     'the TR will be extracted from the '
+                     'header of the fMRI nifti file.')
+    melodic_dir = Directory(exists=True, argstr='-meldir %s',
+                           desc='path to MELODIC directory if MELODIC has already been run')
+    mat_file = File(exists=True, mandatory=True,
+                   argstr='-affmat %s', xor=['feat_dir'],
+                   desc='path/name of the mat-file describing the '
+                   'affine registration (e.g. FSL FLIRT) of the '
+                   'functional data to structural space (.mat file)')
+    fnirt_warp_file = File(exists=True, mandatory=True,
+                          argstr='-warp %s', xor=['feat_dir'],
+                          desc='File name of the warp-file describing '
+                          'the non-linear registration (e.g. FSL FNIRT) '
+                          'of the structural data to MNI152 space (.nii.gz)')
+    motion_parameters = File(exists=True, mandatory=True,
+                            argstr='-mc %s', xor=['feat_dir'],
+                            desc='motion parameters file')
+    denoise_type = traits.Enum('nonaggr', 'aggr', 'both', 'no', usedefault=True,
+                              mandatory=True, argstr='-den %s',
+                              desc='Type of denoising strategy: '
+                              '-none: only classification, no denoising '
+                              '-nonaggr (default): non-aggresssive denoising, i.e. partial component regression '
+                              '-aggr: aggressive denoising, i.e. full component regression '
+                              '-both: both aggressive and non-aggressive denoising (two outputs)')
+
+class ICA_AROMAOutputSpec(TraitedSpec):
+    aggr_denoised_file=File(exists=True, 
+                           desc='if generated: aggressively denoised volume')
+    nonaggr_denoised_file=File(exists=True,
+                              desc='if generated: non aggressively denoised volume' )
+    out_dir = Directory(exists=True, 
+                       desc='directory contains (in addition to the denoised files): '
+                       'melodic.ica + classified_motion_components + '
+                       'classification_overview + feature_scores + melodic_ic_mni)')
+   
+class ICA_AROMA(CommandLine):
+    """
+
```

Comment:
Should describe what this interface does and link to external documentation. Have a look at some other interfaces for examples. You can largely copy this from any existing ICA_AROMA documentation.